### PR TITLE
feat: custom status and latency display

### PR DESCRIPTION
**Pull request by Mantouisyummy**
🚀feat: custom status and latency display

### CODE REVIEW

**Review by wolf-yuan-6115 — APPROVED** [review]
LGTM

File: main.py [hunk 1]
```diff
@@ -34,21 +39,41 @@ def main():
...
+    return activity
+
+
 def setup_logging():
     """
     Set up the loggings for the bot
     :return: None
     """
     formatter = ColoredFormatter(
-        '%(asctime)s %(log_color)s [%(levelname)s] %(message)s',
```
Comment (Nat1anWasTaken) [inline main.py:43]: Can you make these back to ' so the formats keeps the same?

**Review by Nat1anWasTaken — CHANGES_REQUESTED** [review]
TYSM for the PR! I've issued some format changes.

File: main.py (hunk 1 above)
Comment (Mantouisyummy) [reply main.py:43]: Got it. I will change later.

**Review by Nat1anWasTaken — APPROVED** [review]
LGTM!

**Pull request merged**